src/main/java/com/kosherjava/zmanim/hebrewcalendar/YomiCalculator.java

```diff
@@ -97,32 +97,30 @@ public static Daf getDafYomiBavli(JewishCalendar jewishCalendar) {
 
 		int total = 0;
 		int masechta = -1;
-		int blatt = 0;
+		int blatt;
 
 		// Fix Shekalim for old cycles.
 		if (cycleNo <= 7) {
 			blattPerMasechta[4] = 13;
-		} else {
-			blattPerMasechta[4] = 22; // correct any change that may have been changed from a prior calculation
 		}
```

Comment:
The comment says that we need to reassign the value because it might have been changed before. However, the blattPerMasechta variable is recreated every time the method is called. There should be no need for this.